site/reference/pkg/init/README.md

```diff
@@ -3,31 +3,43 @@ title: "Init"
 linkTitle: "init"
 type: docs
 description: >
-   Initialize an empty package
+   Initialize an empty package.
 ---
 <!--mdtogo:Short
-    Initialize an empty package
+    Initialize an empty package.
 -->
 
-{{< asciinema key="pkg-init" rows="10" preload="1" >}}
+`init` initializes an existing empty directory as an empty kpt package.
```

Comment:
it would be more convenient to create the directory if it doesn't exist saving a `mkdir`.

Reply:
yeah, I think that makes sense. I see the the init cobra command needs some cleanup while doing it, so I'll address that separately. I have created an issue: https://github.com/GoogleContainerTools/kpt/issues/1835